src/test/java/tests/TestChangePassword.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import common.Log;
+import helpers.DataHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.ChangePasswordPage;
+import page_objects.HomePage;
+import page_objects.LoginPage;
+import page_objects.RegisterPage;
+
+public class TestChangePassword extends TestBase {
+    HomePage homePage = new HomePage();
+    RegisterPage registerPage = new RegisterPage();
+    LoginPage loginPage = new LoginPage();
+    ChangePasswordPage changePasswordPage = new ChangePasswordPage();
+
+    @Test(description = "User can change password")
+    public void C09() {
```

Comment:
TC09

Reply:
I fixed it. Please check it out for me. Thank you.